Bottom: 56c0d6b251b80d02ee4175ed5c0528e5b47f4935
Top:    9a606c463f7397a2e21ba0f3562615df429cc31a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 15:48:59 +0000

Refresh of tag-circular-contours-so-they-

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 49ee5f4..cb53763 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -625,8 +625,7 @@ do_hole (const BoxType *b, void *cl)
    * hidgl_fill_circle to draw this contour.
    */
   if (curc->is_round) {
-    double slices = M_PI * 2 * curc->radius /
-                    info->scale / PIXELS_PER_CIRCLINE;
+    double slices = calc_slices (curc->radius / info->scale, 2 * M_PI);
     if (slices < curc->Count) {
       hidgl_fill_circle (curc->cx, curc->cy, curc->radius, info->scale);
       return 1;
